src/components/ConductCard/ConductCard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Button, Card, Col, Form} from 'react-bootstrap';
+
+const ConductCard = ({onConductChange, readOnly, cardHeader, conduct, showSaveButton, onConductSaved, saveDisabled}) =>
+{
+    const conductChangeHandler = (event) => {
+        onConductChange(event.target.value);
+    };
+
+    return (
+        <Card className={readOnly ? "mb-4" : "mb-4 mt-4"}>
+            <Card.Header><strong>{cardHeader}</strong></Card.Header>
+            <Card.Body>
+                <Form>
+                    <Form.Group controlId="exampleForm.ControlTextarea1">
```

Comment:
exampleForm.ControlTextarea1?

Reply:
No idea what the purpose of it was. Removed.